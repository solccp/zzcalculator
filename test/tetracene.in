   18
20
 C     0.000000     0.000000     1.400069
 C     1.212495     0.000000     0.700034
 C     1.212495     0.000000    -0.700035
 C     0.000000     0.000000    -1.400069
 C    -1.212495     0.000000    -0.700035
 C    -1.212495     0.000000     0.700035
 C     2.424991     0.000000    -1.400069
 C     3.637486     0.000000    -0.700035
 C     2.424991     0.000000     1.400069
 C     3.637486     0.000000     0.700034
 C     4.849982     0.000000    -1.400069
 C     4.849982     0.000000     1.400069
 C     6.062477     0.000000     0.700034
 C     6.062477     0.000000    -0.700035
 C     7.274972     0.000000    -1.400069
 C     7.274972     0.000000     1.400069
 C     8.487468     0.000000    -0.700035
 C     8.487468     0.000000     0.700034
