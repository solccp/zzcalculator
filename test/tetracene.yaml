---
structure_id: 1
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [18, 17]
- [15, 14]
- [11, 8]
- [16, 13]
- [12, 10]
- [9, 2]
- [7, 3]
---
structure_id: 2
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [18, 16]
- [17, 15]
- [2, 9]
- [3, 7]
- [12, 13]
- [10, 8]
- [11, 14]
---
structure_id: 3
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [18, 16]
- [17, 15]
- [2, 9]
- [3, 7]
- [12, 10]
- [13, 14]
- [8, 11]
---
structure_id: 4
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [18, 16]
- [17, 15]
- [2, 9]
- [3, 7]
rings:
- [12, 10, 8, 11, 14, 13]
---
structure_id: 5
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [18, 16]
- [17, 15]
- [2, 3]
- [7, 8]
- [11, 14]
- [13, 12]
- [10, 9]
---
structure_id: 6
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [18, 16]
- [17, 15]
- [11, 14]
- [13, 12]
rings:
- [2, 3, 7, 8, 10, 9]
---
structure_id: 7
number_of_atoms: 18
double_bonds:
- [1, 6]
- [5, 4]
- [11, 8]
- [7, 3]
- [2, 9]
- [12, 10]
rings:
- [18, 16, 13, 14, 15, 17]
---
structure_id: 8
number_of_atoms: 18
double_bonds:
- [1, 2]
- [6, 5]
- [4, 3]
- [7, 8]
- [9, 10]
- [12, 13]
- [16, 18]
- [11, 14]
- [15, 17]
---
structure_id: 9
number_of_atoms: 18
double_bonds:
- [7, 8]
- [9, 10]
- [11, 14]
- [15, 17]
- [18, 16]
- [12, 13]
rings:
- [1, 2, 3, 4, 5, 6]
