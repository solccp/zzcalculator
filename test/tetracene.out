 5 + 4 x
 total: 9
